From 256ad97f332010ab6673052fe4e562e5ed2dadd2 Mon Sep 17 00:00:00 2001
From: Siu Kwan Lam <michael.lam.sk@gmail.com>
Date: Mon, 3 Feb 2020 14:44:54 -0600
Subject: [PATCH] Update LTO patch

LTO is the only shared library built by LLVM.  It breaks llvmlite
builds under Windows with the following message:

LINK : fatal error LNK1181: cannot open input file 'LTO-NOTFOUND.obj'
---
 tools/lto/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/lto/CMakeLists.txt b/tools/lto/CMakeLists.txt
index b86e4ab..88e5ba0 100644
--- a/tools/lto/CMakeLists.txt
+++ b/tools/lto/CMakeLists.txt
@@ -20,7 +20,7 @@ set(SOURCES
 
 set(LLVM_EXPORTED_SYMBOL_FILE ${CMAKE_CURRENT_SOURCE_DIR}/lto.exports)
 
-add_llvm_library(LTO SHARED INSTALL_WITH_TOOLCHAIN ${SOURCES} DEPENDS intrinsics_gen)
+add_llvm_library(LTO INSTALL_WITH_TOOLCHAIN ${SOURCES} DEPENDS intrinsics_gen)
 
 install(FILES ${LLVM_MAIN_INCLUDE_DIR}/llvm-c/lto.h
   DESTINATION include/llvm-c
-- 
2.16.2

